<?xml version="1.0"?>
<launch>

  
  <master auto="start"/>

  <arg name="base_global_planner" default="navfn/NavfnROS"/>
  <!--<arg name="base_local_planner" default="dwa_local_planner/DWAPlannerROS"/>-->
  <arg name="base_local_planner" default="teb_local_planner/TebLocalPlannerROS"/>

  
  <!--- Run Move Base -->
  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" >
    <param name="base_global_planner" value="$(arg base_global_planner)"/>
    <param name="base_local_planner" value="$(arg base_local_planner)"/> 
    
    <!--<rosparam file="$(find armadillo_navigation_upgrade)/navigation/config/global_costmap_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find armadillo_navigation_upgrade)/navigation/config/local_costmap_params.yaml" command="load" ns="local_costmap" />
     <rosparam file="$(find armadillo_navigation_upgrade)/navigation/config/costmap_common_params.yaml" command="load" ns="local_costmap"/>
     <rosparam file="$(find armadillo_navigation_upgrade)/navigation/config/costmap_common_params.yaml" command="load" ns="global_costmap"/>
    <rosparam file="$(find armadillo_navigation_upgrade)/navigation/config/move_base_params.yaml" command="load"/>-->

    <param name="local_costmap/width" value="3.0"/>
    <param name="local_costmap/height" value="3.0"/> 
	<remap from="cmd_vel" to="nav_vel" />
        
    <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS" />
    <param name="controller_frequency" value="2.0" />
    </node>

</launch>
